installer/tue-install-impl.bash

```diff
@@ -1366,6 +1366,9 @@ do
         --no-test-depend)
             export TUE_INSTALL_TEST_DEPEND="false"
             ;;
+        --branch*)
+            nobleo-install-info "Usage of --branch is deprecated, switch to --try-branch"
+            ;;&
```

Comment:
What is this `&` doing here?

Reply:
It continues the search in the case. So the branch option after this still works.